colourblind8/colourblind8.py

```diff
@@ -1,61 +1,59 @@
 class Colourblind8:
-    def __ini__(self):
-        '''This is the Constructor of the class'''
-
-
-
-    def set_deutera(self,plot):
-        '''
-        A function that converts a matplotlib plot to a Deuteranopia friendly palette
-        and theme with more readable font size and theme design
-
-        inputs
-        ------
-        plot:  a  matplotlib figure that needs reformatting
-
-        outputs:
-        --------
-        new_plot: this is a plot that has been transformed to a Deuteranopia
-                  vision friendly graph, with changes to the background,
-                  gridlines and colour
-
-        '''
-        return new_plot
-
-    def set_prota(self,plot):
-       '''
-        A function that converts a matplotlib plot to a Protanopia friendly palette
-        and theme with more readable font size and theme design
-
-        inputs
-        ------
-        plot:  a  matplotlib figure that needs reformatting
-
-        outputs:
-        --------
-        new_plot: this is a plot that has been transformed to a Protanopia
-                  vision friendly graph, with changes to the background,
-                  gridlines, and colour
-
-        '''
-        return new_plot
-
-       def set_trita(self,plot):
-       '''
-        A function that converts a matplotlib plot to a Tritanopia friendly palette
-        and theme with more readable font size and theme design
-
-        inputs
-        ------
-        plot: a  matplotlib figure that needs reformatting
-
-        outputs:
-        --------
-        new_plot: this is a plot that has been transformed to a Tritanopia
-                  vision friendly graph, with changes to the background,
-                  gridlines, and colour
-
-        '''
-        return new_plot
-
 
+    def __init__(self):
+        self.deutera_colours =  ['#666E95', '#FFC13D', '#2F86E5', '#ACA6C3', '#483E3C',
```

Comment:
These colours have been checked and I think the alternating shades are a good idea.